src/pages/josungmin/Login/Login.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import './Login.scss';
+import './common.scss';
+
+class Login extends React.Component {
+  goToMain = () => {
+    this.props.history.push('/mainsm');
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      password: '',
+      buttonColor: false,
+    };
+  }
+  handleIdPwInput = e => {
+    if (e.target.type === 'text') {
+      this.setState({ id: e.target.value }, this.validation);
+    } else if (e.target.type === 'password') {
+      this.setState({ password: e.target.value }, this.validation);
+    }
```

Comment:
이 부분은 구조분해 할당으로 다음과 같이 작성해볼 수 있을 것 같은데 정상적으로 작동될지 궁금합니다!
id input 태그의 type은 text인데 state의 key값은 id라서 안된다면 name 속성을 state의 key값과 일치하게끔 수정한 뒤, 아래 suggestion에서 type대신 name을 써볼 수도 있을 것 같아요 !

```suggestion
   const {type, value} = e.target;
   this.setState( { [type] : value }, this.validation );
```

Reply:
type으로 어떤 input인지 판단하는건 명확성 측면에서 좋지 않게 생각됩니다(같은 type의 input이 여러개 있을 수 있기에) 유진님 리뷰처럼 name proeprty로 특정시키는게 좋습니다.
그러면 자연스럽게 inputhandler를 하나로 합칠 수도 있겠네요